NEWS

```diff
@@ -1,9 +1,13 @@
-iSubGen 1.0.2 2023-11-06
+iSubGen 1.0.2 2024-08-20
```

Comment:
I need to release a new version as we wrap up the iSubGen for final editing, which is requested by Paul. 
But since the previous version `1.0.2` was not released, so I can keep the version number, right? @dan-knight

Reply:
@dan-knight and I looked into this. I've tagged `v1.0.0` and `v1.0.1` from the corresponding commits. I think you guys double check and we can release them (GH release). They should correspond to the CRAN release as well.

For the next version, we can bundle up `v1.0.2` as Dan's updates didn't change iSubGen itself.